android/app/src/main/java/app/organicmaps/Map.java

```diff
@@ -179,6 +179,7 @@ public void onSurfaceCreated(final Context context, final Surface surface, Rect
         mCallbackUnsupported.report();
       return;
     }
+    sCurrentDpi = surfaceDpi;
```

Comment:
Hm, the check above is not needed anymore?
```
// Do not invoke Framework::UdateDpi after initial surface creation.
if (sCurrentDpi != 0)
``` 
Test this issue https://github.com/organicmaps/organicmaps/issues/6695

Reply:
Yes, this check is not needed. The issue is not reproduced.